packages/venia-ui/lib/components/App/__tests__/__snapshots__/app.spec.js.snap

```diff
@@ -1,5 +1,29 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`displays open nav or drawer 1`] = `
+<div>
+  Title
+  <Main
+    isMasked={false}
+  >
+    <Routes />
+  </Main>
+  <button
+    onClick={[Function]}
+  />
+  <() => {
+  /* istanbul ignore next */
+  cov_101e9trynu().f[0]++;
+  cov_101e9trynu().s[1]++;
```

Comment:
I don't know why code coverage started showing up in _some_ of the snapshots but not others. It has something to do with the jest mock. When the mock is local to the component being tested the coverage comments do not appear.

Reply:
ok it looks like this was happening because I was running with some jest filter like "pattern" or "failed test" which caused the reporter to attach comments for that run. When I run the entire suite, they are removed.